benchmarks/test_emu.py

```diff
@@ -3,6 +3,7 @@
 import pyccl as ccl
 
 EMU_TOLERANCE = 3.0E-2
+cemu = ccl.CosmicemuMTIIPk('tot')
```

Comment:
I think that for these cosmological models there is never a test for the 'cb' case, which is a bit strange to me since we also have cb (and in the other file test_cosmicemu.py there are indeed tests for the 'cb' case). Is this intended?

Reply:
Yeah, this was the original test for the C-implementation, when we were mostly concerned with cosmic shear. I agree we need to test cb too (and hence the new tests!).

Saying this, I'm just realising we don't have a test for cb and tot in the presence of massive neutrinos. Let me create one now.

Reply:
@matteozennaro I've added tests for cosmologies with neutrinos, and I've put the old tests in the same file as the new ones (just so they're all in the same place).